module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,20 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+bool compare(const std::string& s1, const std::string& s2) {
+    if (s1.size() == s2.size()) {
+        return s1 < s2;
+    }
+    return s1.size() < s2.size();
+}
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& lst) {
+    std::deque<std::string> result;
+    for (const auto& el : lst) {
+        result.emplace_back(el);
+    }
```

Comment:
You can initialize deque using list like that:
std::deque<std::string> result{lst.begin(), lst.end()};
So there is no need for emplacing back loop you created.